src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/Symbol.cs

```diff
@@ -16,6 +16,7 @@ internal enum ACCESS
     {
         ACC_UNKNOWN,     // Not yet determined.
         ACC_PRIVATE,
```

Comment:
Might not be done in this PR, but change these to camel-cased without the `ACC_` prefix?

Reply:
At some point I want to make this whole assembly meet with normal C# conventions for naming. I'll probably do so as a wide ranging PR at some point when I (or anyone else) have no PRs since such a change would hit a lot of the code.